CoverMyDotNet.Samples/Program.cs

```diff
@@ -57,16 +57,48 @@ public static void MultipleRequests(Client client)
 			var requests = client.GetRequests(tokens.ToArray());
 			foreach(var v in requests.Requests)
 			{
-				Console.WriteLine("Patient: {0} {1}", v.Patient.FirstName, v.Patient.LastName);
+				Console.WriteLine("Token: {0} {1}", v.Id, v.Tokens.First().Id);
 			}
 		}
+
+		public static void DrugSearch(Client client)
```

Comment:
That would be a good idea. I was mainly using the CoverMyDotNet.Samples to make sure the client class was working correctly with the api. Ill make those changes though.